docs/sdkx-unity/getting-started-android.mdx

```diff
@@ -41,17 +41,16 @@ Guide to integrating the Unity plugin for the Helpshift SDK X which you can call
 <DownloadButton
   text="Download SDK"
   link="[% sdkx-unity.android.sdk_url %]"
-  classes="mb-4"
-/>
+  classes="mb-4"/>
 
 Helpshift SDK _.zip_ folder includes:
 
 <div className="text-sm compact ">
 
-|                                                                      |                                                              |
-| -------------------------------------------------------------------- | ------------------------------------------------------------ |
-| helpshiftX-plugin-unity-[% sdkxunity.package_version %].unitypackage | Unity package of Helpshift SDK                               |
-| unity-jar-resolver (v1.2.170.0)                                      | Resolves Android Helpshift package support lib dependencies. |
+|                                               |             |
+| ----------------------------------------------|-------------|
+| helpshiftX-plugin-unity-v10.3.0.unitypackage | Unity package of HelpshiftX SDK         |
```

Comment:
`HelpshiftX` This is type'o or intentional ?

Reply:
this should be Helpshift SDK X